R/cache.R

```diff
@@ -104,7 +104,7 @@ set_cache <- function(cache_dir = NULL,
                       logfile = NULL,
                       confirm = TRUE) {
   if (is.null(cache_dir) && sessionInfo()$R.version$major >= 4) {
-    cache_dir <- Sys.getenv("EPIDATR_CACHE_DIR", unset = tools::R_user_dir("epidatr"))
+    cache_dir <- Sys.getenv("EPIDATR_CACHE_DIR", unset = rappdirs::user_cache_dir("R", version = "epidatr"))
```

Comment:
this is still behind >= 4 check.  is that intended?

Reply:
also these are different paths.  are they both valid?

Reply:
oh right we prob should have been doing `tools::R_user_dir("epidatr", which = "cache")`

Reply:
Ah ty!